index.bs

```diff
@@ -538,6 +538,7 @@ interface WebTransport {
   readonly attribute WebTransportReliabilityMode reliability;
   readonly attribute WebTransportCongestionControl congestionControl;
   readonly attribute Promise&lt;WebTransportCloseInfo&gt; closed;
+  readonly attribute Promise&lt;WebTransportCloseInfo&gt; draining;
```

Comment:
I thought we agreed that `draining` should not carry CloseInfo, since there is nothing associated in the protocol?

Reply:
Right. Fixing.